### PR TITLE
Clean up ChangeTracking files and fix test failure

### PR DESCRIPTION
**Pull request by gauravga**
@Microsoft/omsagent-devs

### CODE REVIEW

File: installer/datafiles/base_omsagent.data [hunk 1]
```diff
@@ -209,6 +209,10 @@ if ${{PERFORMING_UPGRADE_NOT}}; then
         rmdir /etc/opt/microsoft/omsagent/$ws_id/certs/ 2> /dev/null
         rmdir /etc/opt/microsoft/omsagent/$ws_id/conf/ 2> /dev/null
         rm /opt/microsoft/omsagent/bin/omsagent-$ws_id 2> /dev/null		
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/Changetrackinginventory.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/LinuxFileChangeTracking.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/$ws_id/tmp/ServiceChangeTrackingInventory.xml* 2> /dev/null
+        rm /var/opt/microsoft/omsagent/tmp/*.xml 2> /dev/null
```
Comment (lagalbra) [inline installer/datafiles/base_omsagent.data]: This path should also include `$ws_id`. If you're looking to delete files that should only be in the primary workspace's tmp/ folder, this line should be outside of this workspace-ID loop.

Comment (gauravga) [inline installer/datafiles/base_omsagent.data]: it does

Comment (lagalbra) [inline installer/datafiles/base_omsagent.data]: The last line does not.
You have: `rm /var/opt/microsoft/omsagent/tmp/*.xml 2> /dev/null`
It should be: `rm /var/opt/microsoft/omsagent/**$ws_id**/tmp/*.xml 2> /dev/null`

Comment (NarineM) [inline installer/datafiles/base_omsagent.data]: /var/opt/microsoft/omsagent/tmp/ is just a symbolic link to /var/opt/microsoft/omsagent/wid/tmp for which you already have rm so you can probably remove the last line.
 How about hash files under tmp?

**Pull request merged**